README.md

```diff
@@ -6,21 +6,44 @@ such as `rustc`, `cargo`, `rustup`, etc., as well as additional plugins and targ
 * `cargo-chef` - a tool for building and caching dependencies for a rust project
 * `cargo-sbom` - a tool for generating SBoMs for rust-based projects
 * targets to build for linux amd64, arm64 and riscv64 on linux for both musl and glibc
+* `mold` - a very fast linker for cross-compilation targets
+* `clang` - to invoke `mold` for cross-compilation targets
```

Comment:
These aren't rust plugins and targets, there are generally installed tools. Do they go on this list?

Reply:
> These aren't rust plugins and targets, there are generally installed tools. Do they go on this list?

I'll create a separate section